netops_demo/README.md

```diff
@@ -184,11 +209,10 @@ Modify the source code and build the images:
 NETOPS_TAG=latest make
 ```
 
-This will output `netops-demo-golang:latest` and `netops-demo-py:latest` using Nuclio's ability to [build function images from Dockerfiles](https://github.com/nuclio/nuclio/blob/master/docs/tasks/deploy-functions-from-dockerfile.md). 
-> The `golang` image contains the `ingest` and `query` functions. The `py` image contains the `generate` and `train` functions. By bunching together a few functions inside a single image we allow for easily sharing code without worrying about versioning, reducing the number of moving parts, etc. 
+This will output `netops-demo-golang:latest` and `netops-demo-py:latest` using Nuclio's ability to [build function images from Dockerfiles](https://github.com/nuclio/nuclio/blob/master/docs/tasks/deploy-functions-from-dockerfile.md).
+> The `golang` image contains the `ingest` and `query` functions. The `py` image contains the `generate` and `train` functions. By bunching together a few functions inside a single image we allow for easily sharing code without worrying about versioning, reducing the number of moving parts, etc.
 
 Push the images to your favorite Docker registry:
 ```
 NETOPS_TAG=latest NETOPS_REGISTRY_URL=mydockerhubaccount make push
 ```
-
```

Comment:
There are tons of binary / submodule files added in this PR - please check the changed files and make sure only required files are in